hazenlib/tasks/acr_geometric_accuracy.py

```diff
@@ -85,97 +84,92 @@ def run(self) -> dict:
 
         return results
 
-    def get_geometric_accuracy_slice1(self, dcm):
-        img = dcm.pixel_array
-
-        mask = self.ACR_obj.get_mask_image(self.ACR_obj.images[6])
-        cxy = self.ACR_obj.centre
-        length_dict = self.ACR_obj.measure_orthogonal_lengths(mask)
-
-        if self.report:
-            import matplotlib.pyplot as plt
-            fig, axes = plt.subplots(3, 1)
-            fig.set_size_inches(8, 24)
-            fig.tight_layout(pad=4)
-
-            axes[0].imshow(img)
-            axes[0].scatter(cxy[0], cxy[1], c='red')
-            axes[0].set_title('Centroid Location')
-
-            axes[1].imshow(mask)
-            axes[1].set_title('Thresholding Result')
-
-            axes[2].imshow(img)
-            axes[2].arrow(length_dict['Horizontal Extent'][0], cxy[1],
-                          length_dict['Horizontal Extent'][-1] - length_dict['Horizontal Extent'][0], 1, color='blue',
-                          length_includes_head=True, head_width=5)
-            axes[2].arrow(cxy[0], length_dict['Vertical Extent'][0], 1, length_dict['Vertical Extent'][-1] -
-                          length_dict['Vertical Extent'][0], color='orange', length_includes_head=True, head_width=5)
-            axes[2].legend([str(np.round(length_dict['Horizontal Distance'], 2)) + 'mm',
-                            str(np.round(length_dict['Vertical Distance'], 2)) + 'mm'])
-            axes[2].axis('off')
-            axes[2].set_title('Geometric Accuracy for Slice 1')
-
-            img_path = os.path.realpath(os.path.join(self.report_path, f'{self.img_desc(dcm)}.png'))
-            fig.savefig(img_path)
-            self.report_files.append(img_path)
-
-        return length_dict['Horizontal Distance'], length_dict['Vertical Distance']
 
-    def get_geometric_accuracy_slice5(self, dcm):
-        img = dcm.pixel_array
-        mask = self.ACR_obj.get_mask_image(self.ACR_obj.images[6])
-        cxy = self.ACR_obj.centre
+    def get_geometric_accuracy(self, slice_index):
```

Comment:
this is amazing and very useful for future code maintenance that you combined these functions!